exercises/practice/clock/.docs/instructions.md

```diff
@@ -5,3 +5,11 @@ Implement a clock that handles times without dates.
 You should be able to add and subtract minutes to it.
 
 Two clocks that represent the same time should be equal to each other.
+
+You will also need to impliment `.to_string()` for the `Clock` struct. We will be using this to display the Clock's data.  You can either do it via implimenting  `String::from` or using the [Display trait](https://doc.rust-lang.org/std/fmt/trait.Display.html).
```

Comment:
Instead of "the clock's data", you could consider saying "the clock's state". The interface this exercise expects is an intentional choice intended to allow students to choose whatever data structure makes sense to them for the clock.

`impl From<Clock> for String` will _not_ provide a `to_string` method; that is wrong and needs to be changed. The two most common ways to implement a `to_string` method are via the `Display` trait, or via direct implementation:

```rust
impl Clock {
    fn to_string() -> String { ... }
}
```